src/krate.rs

```diff
@@ -547,13 +549,28 @@ impl Crate {
         }
     }
 
+    // Return None if the documentation URL host matches a blacklisted host
     fn remove_blacklisted_documentation_urls(url: Option<String>) -> Option<String> {
-        let blacklisted_url = "rust-ci.org";
+        let url = match url {
+            Some(url) => url,
+            None => return None,
+        };
+
+        // Handles the case if the documentation URL does not parse successfully
+        let parsed_url = match Url::parse(&url) {
+            Ok(parsed_url) => parsed_url,
+            Err(_) => return None,
+        };
```

Comment:
I did try something fancier involving `if let`, but it was heading to an arrow anti-pattern and you can't have an `if let` statement with the second variable depending on the result of the first. This solution is a lot simpler :).